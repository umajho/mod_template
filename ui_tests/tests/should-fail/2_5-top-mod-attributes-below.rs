fn main() {}

#[mod_template::define]
#[an_attribute]
mod __ {}
