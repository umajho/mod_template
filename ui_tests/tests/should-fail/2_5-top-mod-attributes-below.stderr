error: attributes directly below the attribute `mod_template::define` are not allowed
 --> tests/should-fail/2_5-top-mod-attributes-below.rs:3:1
  |
3 | #[mod_template::define]
  | ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `mod_template::define` (in Nightly builds, run with -Z macro-backtrace for more info)
